modules/cats/src/smithy4s/interopcats/SchemaVisitorShow.scala

```diff
@@ -169,4 +169,12 @@ final class SchemaVisitorShow(
     a => ss.value.show(a)
   }
 
+  override def nullable[A](schema: Schema[A]): Show[Option[A]] = {
+    val showA = self(schema)
+    locally {
+      case None        => "N/A"
+      case Some(value) => showA.show(value)
+    }
```

Comment:
**issue:** still got a problem with this. If you're showing a String with the contents `N/A`, you won't be able to tell whether it's present or null.

I would again suggest wrapping the underlying string in `Some`.

Reply:
_sigh_ fine !